examples/exampleFMU/exampleFMU2_MSRE_HX.f90

```diff
@@ -0,0 +1,167 @@
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!                          Futility Development Group                          !
+!                             All rights reserved.                             !
+!                                                                              !
+! Futility is a jointly-maintained, open-source project between the University !
+! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+! can be found in LICENSE.txt in the head directory of this repository.        !
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!> @brief Program to demo the Transform FMU model of an MSR heat exchanger.
+!> contact Scott Greenwood for additional information on the Transform FMU model.
+!>
+!> To use, Download the example third party FMU from the fmu-ctf-cross-check repo on gitlab:
+!>   wget https://code-int.ornl.gov/wgurecky/fmu-ctf-cross-check/fmus/Transform_v7.fmu
+!>
+!> Extract the FMU with unzip:
+!>   unzip -d /path/to/fmu/transform_example_fmu Transform_v7.fmu
+!>
+!> Run the example program:
+!>   ./Futility_exampleFMU2_MSRE_HX.exe /path/to/fmu/transform_example_fmu
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+PROGRAM testFMU2
+
+  USE Strings
+  USE IntrType
+  USE ParameterLists
+  USE FMU_Wrapper
+  IMPLICIT NONE
+
+  TYPE(FMU2_Slave) :: test_fmu2_slave
+  TYPE(ParamType) :: FMU_params
+  CHARACTER(len=256) :: unzipDirectory
+  INTEGER(SIK) :: id=3_SIK
+  REAL(SRK) :: dt=1.0E-1_SRK
+  REAL(SRK) :: time=0.0_SRK
+  REAL(SRK) :: timeStart=0.0_SRK
+  REAL(SRK) :: timeEnd=1.0E9_SRK
+  REAL(SRK) :: tol=1.0E-7_SRK
+  INTEGER(SIK) :: i
+  ! Storage for in/out vars
+  REAL(SRK) :: T_out, P_out, mflow_out
+  REAL(SRK) :: T_in_out, mflow_pump
+  ! --- Outputs from FMU model ---
+  TYPE(StringType) :: T_out_name
+  TYPE(StringType) :: P_out_name
+  TYPE(StringType) :: mflow_out_name
+  ! --- Inputs from CTF ---
+  TYPE(StringType) :: T_in_name
+  TYPE(StringType) :: P_in_name
+  TYPE(StringType) :: mflow_in_name
+  TYPE(StringType) :: mflow_pump_name
+  TYPE(StringType) :: junk_var_name
+  T_out_name="T_out"
+  P_out_name="P_out"
+  mflow_out_name="mflow_out"
+  T_in_name="T_in"
+  P_in_name="P_in"
+  mflow_in_name="mflow_in"
+  mflow_pump_name="mflow_pump"
+  junk_var_name="dummy_junk"
+
+  IF (IARGC()==1) THEN
+    CALL getarg(1, unzipDirectory)
```

Comment:
Removed `exampleFMU2_MSRE_HX.f90` from the repo.  `exampleFMU2_bouncing_ball.f90` and `exampleFMU2_rectifier.f90` will remain.  I will add checks against the "gold" outputs available at: https://github.com/modelica/fmi-cross-check/tree/master/fmus/2.0/cs/linux64/MapleSim/2016.2/Rectifier  and https://github.com/modelica/fmi-cross-check/tree/master/fmus/2.0/cs/linux64/Test-FMUs/0.0.2/BouncingBall .  I am hesitant to convert these examples into integration tests that run on the CI because it introduces a dependence on downloaded third party FMUs (binaries) which may not be portable.